src/net/server/mod.rs

```diff
@@ -188,6 +171,37 @@ impl MpcHelperServer {
     }
 }
 
+async fn spawn_server<A>(
```

Comment:
Moved this because `start_on()` crept over the clippy limit for lines of code in a function.